hack/gen-flavors.sh

```diff
@@ -17,10 +17,17 @@
 set -o errexit
 set -o nounset
 set -o pipefail
-#set -o verbose
+# set -o verbose
 
 root=$(dirname "${BASH_SOURCE[0]}")/..
+script_invocation_dir="$(pwd)"
 kustomize="${root}/hack/tools/bin/kustomize"
+
+# go to root and install kustomize if missing
+cd ${root} && make "${kustomize##*/}"
+
+# come back to script's invocation dir
+cd ${script_invocation_dir}
```

Comment:
You can actually do `cd -` to return to the previous directory here.

Reply:
Yeah, I didn't have to make use of `cd -` since `make --directory=` served the purpose I was looking for. Thanks for the suggestion!